app/src/main/java/com/x8bit/bitwarden/data/platform/manager/model/FlagKey.kt

```diff
@@ -132,6 +132,15 @@ sealed class FlagKey<out T : Any> {
         override val isRemotelyConfigured: Boolean = true
     }
 
+    /**
+     * Data object holding the feature flag key for the Cipher Key Encryption feature.
+     */
+    data object CipherKeyEncryption : FlagKey<Boolean>() {
+        override val keyName: String = "cipher-key-encryption"
+        override val defaultValue: Boolean = true
+        override val isRemotelyConfigured: Boolean = true
```

Comment:
Should this be off right now, usually we do not turn this on until the feature is 100% ready?

Reply:
The feature was already active if server version was at least "2024.2.0".